charts/stash/templates/apiregistration.yaml

```diff
@@ -99,7 +99,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: {{ template "stash.fullname" . }}-apiserver-extension-server-authentication-reader
-  namespace: kube-system
+  namespace: {{ .Release.Namespace }}
```

Comment:
This is a Role binding and can only created in the namespace of the role which is kube-system. Please revert this change.